docs/orbs-cd.md

```diff
@@ -0,0 +1,69 @@
+# CD Techniques
+
+_Placeholder for more information regarding CD_
+
+## The `circleci/orb-tools` orb:
+
+In order to aid you in your CD efforts we have put together an orb-tools orb you can
+reference while building your own orb.
+
+Find the full source [here](https://github.com/CircleCI-Public/circleci-orbs/blob/master/src/orb-tools/orb.yml)
```

Comment:
Can you please put a comment in this file that says (this is a comment for the docs team when they migrate this):
# TODO: this should point to the registry docs once that is live